src/app/components/new-category/new-category.component.scss

```diff
@@ -0,0 +1,125 @@
+.flex-center {
```

Comment:
As I see, this is the exactly scss used for category-card(whole file). Please keep just one file of scss and refer both TS components (category-card and new-category) to that one scss file